extensions/amp-carousel/0.1/slidescroll.js

```diff
@@ -410,16 +410,16 @@ export class AmpSlideScroll extends BaseSlides {
     this.snappingInProgress_ = true;
     const newIndex = this.getNextSlideIndex_(currentScrollLeft);
     this.vsync_.mutate(() => {
-      //TODO (camelburrito): Identify more platforms that dont require
+      //TODO (camelburrito): Identify more platforms that require
       // -amp-no-scroll.
-      if (!this.isAndroidFF_) {
+      if (this.isIos_) {
```

Comment:
what I mean is that I am confused as to what stops the scrolling on Chrome/Android now (since Chrome does momentum scrolling on Android) but it does seem to stop at correct boundaries without  `no-scroll` but why?